content/metadata.yaml

```diff
@@ -24,5 +24,13 @@ authors:
       - Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
       - Related Sciences
     funders: GBMF4552
+  - github: zietzm
+    name: Michael Zietz
+    initials: MZ
+    orcid: 0000-0003-0539-630X
+    twitter: ZietzMichael
+    email: michael.zietz@gmail.com
+    affiliations:
+      - Department of Biomedical Informatics, Columbia University, New York, New York, United States of America
```

Comment:
It might make sense to also add your prior UPenn affiliation as a second affiliation.

Reply:
Good call, thanks!